wolfSSL/src/chapter02.md

```diff
@@ -472,17 +604,53 @@ Turns on the use of DTLS, or datagram TLS. This isn't widely supported or used.
 
 Turns on wolfSSL’s RSA key generation functionality. See [Keys and Certificates](chapter07.md#keys-and-certificates) for more information.
 
-#### WOLFSSL_RIPEMD
+#### WOLF_PRIVATE_KEY_ID
+
+This is used with PKCS11 to enable support for key ID and label API's. FIPS v5 and older doesn't support `WOLF_PRIVATE_KEY_ID` with PK callbacks.
+
+#### WOLFSSL_WOLFSENTRY_HOOKS
+
+Enables wolfSentry hook support.
+
+#### WOLFSSL_CERT_EXT
+
+Certificate extension, key and cert generation feature.
+
+#### WOLFSSL_CERT_REQ
+
+Certificate request, key, and cert generation feature.
+
+#### WOLFSSL_SSLKEYLOGFILE
+
+This enables the key logging used by Wireshark. It does produce a compiler warning since the master secret and client random are written to a file. This is useful for testing and not recommended for production.
+
+#### WOLFSSL_SSLKEYLOGFILE_OUTPUT
+
+This macro defines the filename for the key logging. It is used with `WOLFSSL_SSLKEYLOGFILE`.
+
+#### WOLFSSL_HAVE_WOLFSCEP
 
-Enables RIPEMD-160 support.
+Enable feature used by autoconf to see if wolfSCEP is available.
 
-#### WOLFSSL_SHA384
+#### WOLFSSL_HAVE_MIN
 
-Enables SHA-384 support.
+This macro is for portability of the library to indicate if MIN/MAX are already defined by the platform. It prevents duplicate definitions.
 
-#### WOLFSSL_SHA512
+#### WOLFSSL_HAVE_TLS_UNIQUE
 
-Enables SHA-512 support.
+Added in libest port: allow applications to get the 'tls-unique' Channel Binding Type (https://tools.ietf.org/html/rfc5929#section-3). This is used in the EST protocol to bind an enrollment to a TLS session through 'proof-of-possession' (https://tools.ietf.org/html/rfc7030#section-3.4 and https://tools.ietf.org/html/rfc7030#section-3.5).
+
+#### WOLFSSL_ENCRYPTED_KEYS
+
+Enable for encrypted keys PKCS8 support.
```

Comment:
Add that it enables PKCS8 password based key encryption. Might be helpful to add link to RFC https://datatracker.ietf.org/doc/html/rfc5208